src/components/UI/Text/InputNumber.vue

```diff
@@ -263,9 +263,8 @@ export default {
     keyPress(event) {
       // invoked when a key is pressed
       // character key not accepted
-      event = event || window.event;
-      var charCode = (typeof event.which == "undefined") ? event.keyCode : event.which;
-      var charStr = String.fromCharCode(charCode);
+      let charCode = event.keyCode;
```

Comment:
These things should always be checked. 
`keyCode` is deprecated
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Reply:
will use event.key will work the same

Reply:
will use event.key will work the same